core/src/utils/overlays.ts

```diff
@@ -685,6 +685,7 @@ export const safeCall = (handler: any, arg?: any) => {
 
 export const BACKDROP = 'backdrop';
 export const GESTURE = 'gesture';
+export const OVERLAY_GESTURE_PRIORITY = 39;
```

Comment:
I like this approach - is the idea that future gesture priorities will be abstracted into this file?

Reply:
I don't know if it would makes sense to put all the gesture priorities in this file specifically, but I think long term it would be good to have them consolidated somewhere. Right now it's difficult to understand if changing a priority will cause other gestures used at the same time to break. Some of the gesture priorities seem pretty arbitrary, so I'd like to add some documentation around why they were chosen.